.github/workflows/deploy-to-environment.yml

```diff
@@ -13,22 +13,23 @@ on:
         options:
           - staging
           - beta
-        default: staging
```

Comment:
why removing the defaults?

Reply:
I added the defaults in an earlier attempt to fix to run-name. Thought that by adding them, the run name would get those defaults. Didn't work so no real need.

Reply:
thank you for the explanation